web/src/components/apps/AppVersionHistory.tsx

```diff
@@ -240,7 +238,7 @@ class AppVersionHistory extends Component<Props, State> {
     this.state.appUpdateChecker.start(this.getAppUpdateStatus, 1000);
 
     const url = window.location.pathname;
-    const { params } = this.props.match;
+    const { params } = this.props.wrappedMatch;
```

Comment:
I could not figure out what was going on with the `match` for this component but it's getting modified somewhere.